### PR TITLE
cssgrids

### PR DESCRIPTION
**Pull request by Pace675**

### CODE REVIEW

File: src/css-grids/css-grids.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CSS-Grids</title>
+    <link rel="stylesheet" href="css-grids-styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower&display=swap" rel="stylesheet">
```
Comment (juan-rod) [inline src/css-grids/css-grids.html:8]: bump this font link up before your css styles

File: src/css-grids/css-grids.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CSS-Grids</title>
+    <link rel="stylesheet" href="css-grids-styles.css">
```
Comment (juan-rod) [inline src/css-grids/css-grids.html]: add `./` before the href= link

File: src/css-grids/css-grids.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+          <div class="class button-element">=</div>
+        </div>
+        <div class="button-row">
+          <div class="class button-element">1</div>
+          <div class="class button-element">2</div>
+          <div class="class button-element">3</div>
+          <div class="class button-element">0</div>
+      </div>
+      <style style="text/css">
+        .scroll-up {
```
Comment (juan-rod) [inline src/css-grids/css-grids.html]: i'm assuming this is something other than the grid styling. put it in it's own css stylesheet and add the link to the head element on this page

File: src/nasas.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+<body>
+  <main>
+    <h1>NASA</h1>
+  </main>
+  <section>
+    <article>Social Media</article>
+    <p class="sub1">Twitter - @nasa</p>
+    <p class="sub1">Facebook - fb.com/nasa</p>
+    <p class="sub1">Instagram - @nasa</p>
+    <p class="sub1">Youtube - youtube.com/nasa</p>
```
Comment (juan-rod) [inline src/nasas.html]: we JUST talked about this! yes. wrap these in their own divs that way section flexbox will just affect three divs: social-media, office info, contact us.
`<div class="social">`
    ` <article>Social Media</article>`
     `<p class="sub1">Twitter - @nasa</p>`
     `<p class="sub1">Facebook - fb.com/nasa</p>`
      `<p class="sub1">Instagram - @nasa</p>`
      `<p class="sub1">Youtube - youtube.com/nasa</p>`
    `</div>`

File: src/project/project.html [hunk 5]
```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Check Point 1</title>
+  <link rel="stylesheet" href="./css/reset.css">
```
Comment (juan-rod) [inline src/project/project.html:8]: is this your final checkpoint?

**Review by juan-rod — APPROVED** [review]
great work! tons of progress! I see a lot of unnecessary code and makes me think you're getting it from somewhere and trying it out. That's completely fine! But don't rely on doing that too much!
also, is your checkpoint in this branch? 
merge when ready

File: src/project/project.html (hunk 5 above)
Comment (Pace675) [reply src/project/project.html:8]: Yes

File: src/nasas.html (hunk 4 above)
Comment (Pace675) [reply src/nasas.html]: Done

File: src/css-grids/css-grids.html (hunk 1 above)
Comment (Pace675) [reply src/css-grids/css-grids.html:8]: done

Comment (Pace675) [reply src/css-grids/css-grids.html:8]: done

File: src/css-grids/css-grids.html (hunk 3 above)
Comment (Pace675) [reply src/css-grids/css-grids.html]: I just moved it to the primary css file and it clear alot of clutter in the HTML

**Pull request merged**